example/spec/syncGranule/SyncGranuleSuccessSpec.js

```diff
@@ -38,6 +38,56 @@ describe('The Sync Granules workflow', () => {
     expect(workflowExecution.status).toEqual('SUCCEEDED');
   });
 
+  describe('when configured to handle duplicates as error', () => {
+    let collectionInfo;
+
+    process.env.CollectionsTable = `${config.stackName}-CollectionsTable`;
+    const c = new Collection();
+
+    beforeAll(async () => {
+      collectionInfo = await c
+        .update(collection, { duplicateHandling: 'error' })
+        .then(() => c.get(collection));
+    });
+
+    it('configured collection to handle duplicates as error', () => {
+      expect(collectionInfo.duplicateHandling, 'error');
+    });
+
+    describe('and configured not to catch errors', () => {
+      let failWorkflowExecution;
+
+      beforeAll(async () => {
+        failWorkflowExecution = await buildAndExecuteWorkflow(
+          config.stackName, config.bucket, taskName, collection, provider, inputPayload
+        );
+      });
+
+      it('fails the workflow', () => {
+        expect(failWorkflowExecution.status).toEqual('FAILED');
+      });
+    });
+
+    describe('and configured to catch errors', () => {
+      let catchWorkflowExecution;
+      const catchTaskName = 'SyncGranuleCatchWorkflow';
+
+      beforeAll(async () => {
+        catchWorkflowExecution = await buildAndExecuteWorkflow(
+          config.stackName, config.bucket, catchTaskName, collection, provider, inputPayload
+        );
+      });
+
+      it('completes execution with success status', () => {
+        expect(catchWorkflowExecution.status).toEqual('SUCCEEDED');
+      });
+    });
+
+    afterAll(async () => {
+      await c.update(collection, { duplicateHandling: 'replace' });
```

Comment:
In the spirit of tests cleaning up after themselves, would it make sense to delete the collection from the table here?

Reply:
"the table" meaning DynamoDB

Reply:
So yeah, it seems like @mennovandiermen's work is going to handle deleting the collections: https://github.com/nasa/cumulus/pull/531/files#diff-35598cc4cda420a18e2db37a241b2996R30

Reply:
oh! I didn't realize that these weren't being created out of scope. What you have looks :100: to me, then. Thank you for the clarification.